tests/unit/schema/test.firefox-schemas-import.js

```diff
@@ -1112,58 +1111,88 @@ describe('firefox schema import', () => {
 
   describe('fetchSchemas', () => {
     const outputPath = 'tests/fixtures/schema/imported';
-    const expectedTarballPath = 'tmp/FIREFOX_AURORA_54_BASE.tar.gz';
+    const expectedZipfilePath = 'tmp/FIREFOX_AURORA_54_BASE.zip';
 
     beforeEach(() => {
-      expect(fs.existsSync(expectedTarballPath)).toEqual(false);
+      expect(fs.existsSync(expectedZipfilePath)).toEqual(false);
       createDir(outputPath);
     });
 
     afterEach(() => {
-      expect(fs.existsSync(expectedTarballPath)).toEqual(false);
+      expect(fs.existsSync(expectedZipfilePath)).toEqual(false);
       removeDir(outputPath);
     });
 
     it('extracts the schemas from a local file', async () => {
       const cwd = 'tests/fixtures/schema';
-      const schemaPath = 'firefox';
-      const tarball = tar.create({ cwd, gzip: true }, [schemaPath]);
+      const schemaPath = `${cwd}/firefox`;
+      const zipfile = new yazl.ZipFile();
+
+      const files = [
+        'cookies.json',
+        'manifest.json',
+        'native_host_manifest.json',
+      ];
+
+      await new Promise((resolve) => {
+        files.forEach((file) => {
+          zipfile.addFile(`${schemaPath}/${file}`, file);
+        });
+        zipfile.outputStream
+          .pipe(fs.createWriteStream('mozilla-central.zip'))
+          .on('close', () => resolve());
+        zipfile.end();
+      });
+
```

Comment:
Nits:
- I think that we zipfile.outputStream is the only part that really need to be wrapped in a promise
- it seems that we are using similar code in at least one other test in this same test file, how about create a small test helper and reuse it in both places?

Reply:
Indeed, fixed in cf90083b

Reply:
Added a small cleanup in 06b0863b - the `zipfile.end()` needs to be called inside the promise, the `addFile` calls don't.

Reply:
I guess that the promise that resolves when the zipfile.outputStream fires the "close" event should be created before calling `zipfile.end()` to ensure that the promise is ever resolved, but anyway calling zipfile.end() inside the promise is also fine as well.